lib/rspec/core/configuration.rb

```diff
@@ -711,15 +711,7 @@ def color_enabled?(output=output_stream)
       # @attr true_or_false [Boolean] toggle color enabled
       def color=(true_or_false)
         return unless true_or_false
-
-        if RSpec::Support::OS.windows? && !ENV['ANSICON']
-          RSpec.warning "You must use ANSICON 1.31 or later " \
-                        "(http://adoxa.3eeweb.com/ansicon/) to use colour " \
-                        "on Windows"
-          @color = false
-        else
-          @color = true
-        end
+        @color = !!true_or_false
       end
```

Comment:
Sounds good.  I didn't know if that method needed to hang around or not.  I'll change it to ```attr_writer :color```